src/PAModel/Themes/Theme.cs

```diff
@@ -46,6 +46,10 @@ private void LoadTheme(CustomThemeJson theme)
                 var styleName = style.name;
                 foreach (var prop in style.propertyValuesMap)
                 {
+                    if(prop.value == ""){
```

Comment:
Should the ErrorContainer be passed down to this? ArgumentException doesn't seem like the right way to handle this, it's possible this should only be a warning.

Reply:
Basically, I think there might be some apps out there with this condition that are actively in use, since Doc Server doesn't enforce it strongly. It only crashes in some scenarios, I think if the default script is used, or something like that.

Reply:
Okay; I'm currently trying to connect an ErrorContainer to this. Where are the ErrorContainers typically in the lifecycle of the program, so that I can pass it down the code towards this point?

Reply:
There's one created at the start of each operation in CanvasDocument, used for the lifetime of that operation. They should be available in all the places we're calling the Theme constructor

Reply:
Okay, thank you! I've connected the ErrorContainer now :) Now testing